include/ptp_clock.h

```diff
@@ -59,12 +59,12 @@ struct ptp_clock {
 #define PTP_CLOCK_GET(dev_name, sfx)					\
 	((struct ptp_clock *)&PTP_CLOCK_GET_NAME(dev_name, sfx))
 
-#define PTP_CLOCK_INIT(dev_name, sfx, api)				\
+#define PTP_CLOCK_INIT(dev_name, sfx, ptp_api)				\
```

Comment:
so no need of that patch then (since ptp clock dev driver api needs to be reworked.)